workers/wfsfeature-harvester/src/main/java/org/fao/geonet/harvester/wfsfeatures/worker/EsWFSFeatureIndexer.java

```diff
@@ -260,7 +256,10 @@ public void indexFeatures(Exchange exchange) throws Exception {
             BulkResutHandler brh = new AsyncBulkResutHandler(phaser, typeName, url, nbOfFeatures, report);
 
             long begin = System.currentTimeMillis();
-            FeatureIterator<SimpleFeature> features = wfs.getFeatureSource(typeName).getFeatures(query).features();
+
+            ReprojectingFeatureCollection rfc = new ReprojectingFeatureCollection(wfs.getFeatureSource(typeName).getFeatures(query), CRS.decode("urn:ogc:def:crs:OGC:1.3:CRS84"));
```

Comment:
I like this approach, very safe to retroject to something you *know* just based on the returned features.

Reply:
I do not think query is adding value above, it just has query.setCoordinateSystem( ... ) which is then going to be managed locally by ReprojectingFeatureCollection.

Consider simplification to:
```
SimpleFeatureCollection features = was.getFeatureSource(typeName().getFeatures();
ReprojectingFeatureCollection rfc = new ReprojectingFeatureCollection( features, CRS.decode("urn:ogc:def:crs:OGC:1.3:CRS84"));
```

Reply:
So if we not more use the query object, should I also remove the declaration of it ?

Line 248 -> 250

Reply:
Yes good catch!

Reply:
Done